### PR TITLE
Feature/Footer component

### PR DESCRIPTION
**Pull request by hshahwan**
relates #30 
- create 3 buttons (PreviousButton,BackToSecnario,NextButton)
- static component without any functionality

### CODE REVIEW

File: client/src/Component/Common/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import {
+  Buttons,
+  NextButton,
+  PreviousButton,
```
Comment (teenie-quaggard) [inline client/src/Component/Common/Footer/index.js:5]: Just a typo here that may be good to fix

PreviousButton

File: client/src/Component/Common/Footer/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  BackToSecnario,
```
Comment (teenie-quaggard) [inline client/src/Component/Common/Footer/index.js]: Also a typo here 

BackToScenerio

File: client/src/Component/Common/Footer/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+class Footer extends Component {
+  state = {};
+
+  render() {
+    return (
+      <FooterComponent>
+        <Line />
+        <Buttons>
+          <NextButton type="button" />
+          <BackToSecnario>review your scenario</BackToSecnario>
```
Comment (teenie-quaggard) [inline client/src/Component/Common/Footer/index.js]: Just for future bugproofing... here too

File: client/src/Component/Common/Footer/Footer.style.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  width: 5.2rem;
+  clip-path: polygon(50% 0%, 0% 100%, 100% 100%);
+  background: #faa634;
+  transform: rotate(90deg);
+  font-size: 1rem;
+  border: none;
+  border-radius: 0.5rem;
+`;
+
+export const BackToSecnario = styled.button`
```
Comment (teenie-quaggard) [inline client/src/Component/Common/Footer/Footer.style.js]: typo

File: client/src/Component/Common/Footer/Footer.style.js [hunk 5]
```diff
@@ -4,47 +4,52 @@ export const Buttons = styled.div`
...
   justify-content: space-between;
-  padding: 2.5rem;
+  padding: 0.9rem;
 `;
 
 export const FooterComponent = styled.div`
-  margin-top: 40.5rem;
-  position: fixed;
+  height: 70%;
+  margin-top: 440px;
```
Comment (ali-7) [inline client/src/Component/Common/Footer/Footer.style.js]: change to rem 44rem

**Review by ali-7 — CHANGES_REQUESTED** [review]

File: client/src/Component/Common/Footer/index.js (hunk 1 above)
Comment (hshahwan) [reply client/src/Component/Common/Footer/index.js:5]: I think it's the same

File: client/src/Component/Common/Footer/index.js (hunk 2 above)
Comment (hshahwan) [reply client/src/Component/Common/Footer/index.js]: done

File: client/src/Component/Common/Footer/index.js (hunk 3 above)
Comment (hshahwan) [reply client/src/Component/Common/Footer/index.js]: update it

**Review by ali-7 — APPROVED** [review]

**Review by teenie-quaggard — APPROVED** [review]
Thanks Haneen!

**Pull request merged**